brigade.js

```diff
@@ -5,24 +5,20 @@ events.on("push", (e, project) => {
   // this is DinD.
   var driver = project.secrets.DOCKER_DRIVER || "overlay"
 
-  // Build and test in prod configuration.
-  const prodBuild = new Job("prod", "node:8")
-  /*
-  prodBuild.cache.enabled = true
-  prodBuild.cache.path = "/src/node_modules"
-  */
-  prodBuild.tasks = [
+  const unitTests = new Job("dev", "node:8")
+  unitTests.tasks = [
     "cd /src",
     "yarn install",
-    "yarn build"
+    "ng lint",
+    "ng test --single-run",
+    "ng e2e"
   ]
 
-  // Build and test in dev configuration.
-  const devBuild = new Job("dev", "node:8")
-  devBuild.tasks = [
+  const e2e = new Job("dev", "node:8")
+  e2e.tasks = [
     "cd /src",
     "yarn install",
```

Comment:
@adamreese @technosophos is it possible cache this dependency step (`yarn install`) in brigade?